tx-generator-shelley/test/Cardano/Benchmarking/ReferenceServer.hs

```diff
@@ -194,22 +195,22 @@ initialServerState = ServerState 0 Seq.empty Map.empty Map.empty 0
 -- and does not make any delta-Q info to optimises the pipelining decisions.
 --
 txSubmissionServer
-  :: forall txid tx m.
-     (Ord txid, Show txid, Show tx, Monad m)
+  :: forall txid tx m out.
+     (out ~ [ (tx,UTCTime) ], Ord txid, Show txid, Show tx, MonadTime m)
   => Tracer m (TraceEventServer txid tx)
   -> (tx -> txid)
   -> Word16  -- ^ Maximum number of unacknowledged txids
   -> Word16  -- ^ Maximum number of txids to request in any one go
   -> Word16  -- ^ Maximum number of txs to request in any one go
-  -> TxSubmissionServerPipelined txid tx m [tx]
+  -> TxSubmissionServerPipelined txid tx m out
```

Comment:
it's not enough to put the type `[(tx,UTCTime)]` here?

Reply:
That would have worked as well.

Reply:
The same type `[(tx,UTCTime)]` is used in about 7 or 8 places, I thought it was better to name it as `out`. Before the change it was `[tx]` but when changing it I was not sure if every `[tx]` will be replaced with `out`.